Mathlib/Algebra/Polynomial/AlgebraMap.lean

```diff
@@ -304,6 +304,20 @@ theorem algEquivOfCompEqX_eq_iff (p q p' q' : R[X])
 theorem algEquivOfCompEqX_symm (p q : R[X]) (hpq : p.comp q = X) (hqp : q.comp p = X) :
     (algEquivOfCompEqX p q hpq hqp).symm = algEquivOfCompEqX q p hqp hpq := rfl
 
+/-- The automorphism of the polynomial algebra given by `p(X) ↦ p(a * X + b)`,
+  with inverse `p(X) ↦ p(a⁻¹ * (X - b))`. -/
+@[simps!]
+def algEquivCMulXAddC {R} [CommRing R] (a b : R) [Invertible a] : R[X] ≃ₐ[R] R[X] :=
```

Comment:
```suggestion
def algEquivCMulXAddC {R : Type*} [CommRing R] (a b : R) [Invertible a] : R[X] ≃ₐ[R] R[X] :=
```
I think this is the most common spelling in mathlib.

Reply:
Sorry I've forgotten about this PR.
I recall it used to be the case that arguments in a def/lemma can be inferred, but every argument appearing in `variable` must be given a type, which is maybe why people got the habit to specify Type* everywhere. Now in Lean 4, even the `variable`s can have their types inferred, and I see little reason not to make use of this feature.

Reply:
It's not very important, so don't worry about it.

I've asked a [question](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/.60lemma.20foo.20.7BR.7D.60.20or.20.60lemma.20foo.20.7BR.20.3A.20Type*.7D.60), but in the meantime this can be merged.